airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -64,13 +64,13 @@ def split_properties(properties_list: List[str]) -> Iterator[Tuple[str]]:
     summary_length = 0
     local_properties = []
     for property_ in properties_list:
-        if len(property_) + summary_length + len(urllib.parse.quote(",")) >= PROPERTIES_PARAM_MAX_LENGTH:
+        if len(urllib.parse.quote(f"property={property_}&")) + summary_length >= PROPERTIES_PARAM_MAX_LENGTH:
```

Comment:
nit: can we store the length in a variable and reuse it in https://github.com/airbytehq/airbyte/pull/13691/files#diff-6c388abccab93dc7577dc93da2d16f691f60bb9c806ec20011457fc10070b07cR73 to ensure the values stay in sync?

Reply:
sure, done